operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1566,14 +1525,8 @@ memcached:
     # -- Memcached Docker image pull policy
     pullPolicy: IfNotPresent
 
-  # -- The SecurityContext for memcached pods
-  podSecurityContext:
-    fsGroup: 11211
-    runAsGroup: 11211
-    runAsNonRoot: true
-    runAsUser: 11211
-    seccompProfile:
-      type: RuntimeDefault
+  # -- The SecurityContext override for memcached pods
+  podSecurityContext: {}
```

Comment:
The `memcache` user and group inside the memcached alpine container with uid and gid 11211. I tried running this change and it passed ok. I also tested a migration from 11211 to 10001 and it was ok. I don't quite remember why I had the ID the same as the one in the image. Any ideas? If not, we can leave it with 10001

Reply:
I tested it as well, I think it was copy paste from the port number :)